### DISCUSSION TITLE
Hooks support dropped? Any plan to introduce hooks again?

### DISCUSSION THREAD

**Discussion by robert-h-at-cp** [dynamoose/dynamoose#1230] [Q&A]
I can find some discussions that went on for a bit on this topic. And I've went through the related issues, PRs, and chats on Gitter lobby. I'm wondering what the current status is on this feature? It can be a very helpful feature for my current project. And I guess a pretty standard one for most object <x> mapping libraries.

Any info would be appreciated. Thanks

**Comment by fishcharlie** [comment]
@robert-h-at-cp What discussions are you referring to? Are you referring to v1 plugins? Can you provide some example code of what you are looking for? This is a super broad question.

**Reply by robert-h-at-cp** [reply]
@fishcharlie Thanks for looking into this.

I'm referring to the following github issues, PR, and gitter chat.

* https://github.com/dynamoose/dynamoose/pull/9
* https://github.com/dynamoose/dynamoose/issues/244
* https://github.com/dynamoose/dynamoose/pull/325
* https://gitter.im/dynamoosejs/Lobby?at=5a196be4540c78242d489e15

Essentially, I'm looking for something like the Mongoose `pre` & `post` hooks - https://mongoosejs.com/docs/middleware.html#pre

Sorry for not being specific enough.

**Reply by fishcharlie** [reply]
@robert-h-at-cp https://github.com/dynamoose/dynamoose/issues/851

**Reply by robert-h-at-cp** [reply]
Thanks
